packages/react-dom/index.js

```diff
@@ -14,7 +14,6 @@ export {
   createPortal,
   createRoot,
   hydrateRoot,
-  findDOMNode,
```

Comment:
Should we provide an error like `"findDOMNode has been removed, see react.dev/finddomnode for more info"`?